docs/source/quickstart.rst

```diff
@@ -1,22 +1,297 @@
+            },
+            out_col=CONCEPT.PARTICIPANT.ID
+        ),
+        keep_map(
+            in_col="mother",
+            out_col=CONCEPT.PARTICIPANT.MOTHER_ID
+        ),
+        keep_map(
+            in_col="father",
+            out_col=CONCEPT.PARTICIPANT.FATHER_ID
+        ),
+        value_map(
+            in_col="gender",
+            # Don't worry about mother/father gender here.
+            # We can create them in a later phase.
+            m={
+                "F": constants.GENDER.FEMALE,
+                "M": constants.GENDER.MALE
+            },
+            out_col=CONCEPT.PARTICIPANT.GENDER
+        ),
+        value_map(
+            in_col="specimens",
+            m=lambda x: Split(re.split("[,;]", x)),
+            out_col=CONCEPT.BIOSPECIMEN.ID
+        ),
+        [
+            value_map(
+                in_col=6,  # age (hrs) (first)
+                m=lambda x: int(x) / 24,
+                out_col=CONCEPT.PHENOTYPE.EVENT_AGE_DAYS
+            ),
+            melt_map(
+                var_name=CONCEPT.PHENOTYPE.NAME,
+                map_for_vars={
+                    "CLEFT_EGO": "Cleft ego",
+                    "CLEFT_ID": "Cleft id"
+                },
+                value_name=CONCEPT.PHENOTYPE.OBSERVED,
+                map_for_values=observed_yes_no
+            )
+        ],
+        [
+            value_map(
+                in_col=9,  # age (hrs) (second)
+                m=lambda x: int(x) / 24,
+                out_col=CONCEPT.PHENOTYPE.EVENT_AGE_DAYS
+            ),
+            melt_map(
+                var_name=CONCEPT.PHENOTYPE.NAME,
+                map_for_vars={
+                    "EXTRA_EARDRUM": "Extra eardrum"
+                },
+                value_name=CONCEPT.PHENOTYPE.OBSERVED,
+                map_for_values=observed_yes_no
+            )
+        ]
+    ]
+
+It goes in the ``extract_configs`` directory like this::
+
+    my_study/
+    ├── data
+    │   └── clinical.tsv
+    └── extract_configs
+        ├── extract_config.py
+        └── family_and_phenotype.py
+
+Run the ingest pipeline
+=======================
+
+The Warehouse Server
+--------------------
+
+Data that we ingest is meant to be shared, not stored locally on your machine!
+It's easy to configure the ingest system to connect to a remote warehousing
+database which will centrally store the relevant data.
+
+Local Environment Configuration
+-------------------------------
+
+All that's needed is the appropriate database to be setup with the required
+access permissions (see :ref:`Warehousing` for all the details). We just need
+to set the local variable
+``KF_WAREHOUSE_DB_URL`` with a complete login URL::
+
+    export KF_WAREHOUSE_DB_URL=postgresql://<username>:<password>@<address>:<port>/postgres
+
+The final Extraction product
+----------------------------
+
+Now we're ready to perform extraction, which is done with the following
```

Comment:
Let's move this (introducing the ingest command) to the end of this Run ingest pipeline section. And instead of saying `Now we're ready to perform extraction, which is done with the following` can you say something like:

```
If the ingest package passes validation with the test command and you're happy with the output, you're ready ingest the data into your target service using the ingest command::

    $ kidsfirst ingest my_study 

The library defaults to ingesting data into the Kids First Data Service at the base url: `http://localhost:5000`. You can change the base URL for this service using the `--target_url` CLI option.
```